js/kontent.js

```diff
@@ -20,7 +20,7 @@ class URLBuilder {
         if (this.includeTotalCount) searchParams.set("includeTotalCount", this.includeTotalCount);
         if (this.limit) searchParams.set("limit", this.limit);
         if (this.orderByElementProp) searchParams.set("order", `elements.${this.orderByElementProp}`);
-        if (this.blogTagsContains) searchParams.set("elements.blog_tags[contains]", this.blogTagsContains);
+        if (this.blogTagsContains) searchParams.set("elements.blog_tags[any]", this.blogTagsContains);
```

Comment:
is `blogTagsContains` still the right name?

Reply:
`selectedTags` better